.travis.yml

```diff
@@ -0,0 +1,26 @@
+
+language: ruby
+rvm:
+  - ruby-2.2.3
+
+before install:
+  - sudo add-apt-repository --yes ppa:ubuntu-sdk-team/ppa
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq qtdeclarative5-dev qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
+  - export QMAKE=/usr/lib/x86_64-linux-gnu/qt5/bin/qmake
+
+install:
+  - bundle install
+  - export RAILS_ENV=test
+  - bundle exec rake db:create
+  - bundle exec rake db:schema:load
+  - bundle exec rake db:migrate RAILS_ENV=test
```

Comment:
Can remove the RAILS_ENV:
https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables